numpy/core/tests/test_memmap.py

```diff
@@ -126,7 +126,7 @@ def test_arithmetic_drops_references(self):
     def test_indexing_drops_references(self):
         fp = memmap(self.tmpfp, dtype=self.dtype, mode='w+',
                     shape=self.shape)
-        tmp = fp[[(1, 2), (2, 3)]]
+        tmp = fp[((1, 2), (2, 3))]
```

Comment:
Here, the parenthesis are not necessary, correct?

Reply:
Fixed